src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -27,7 +40,32 @@ public static void report(
       MatchPlayer matchPlayer,
       Match match,
       Player player,
-      @Text String reason) {
+      @Text String reason)
+      throws CommandException {
+    if (!commandSender.hasPermission(Permissions.STAFF) && commandSender instanceof Player) {
+      // Check for cooldown
+      Instant lastReport = lastReportSent.getIfPresent(matchPlayer.getId());
+      if (lastReport != null) {
+        Duration timeSinceReport = Duration.between(lastReport, Instant.now());
+        long secondsRemaining = REPORT_COOLDOWN_SECONDS - timeSinceReport.getSeconds();
+        if (secondsRemaining > 0) {
+          throw new CommandException(
+              AllTranslations.get()
+                  .translate(
+                      "command.cooldown",
+                      commandSender,
+                      ChatColor.AQUA
+                          + (secondsRemaining
+                              + " second"
+                              + (secondsRemaining != 1 ? "s" : "")
```

Comment:
This should be translatable.

Reply:
Thanks Molka, I should have done this from the beginning!